.eslintrc.yml

```diff
@@ -6,6 +6,7 @@ extends: 'eslint:recommended'
 globals:
   Atomics: readonly
   SharedArrayBuffer: readonly
+parser: babel-eslint
```

Comment:
eslint :( it understands ES modules but not Symbol ... ?

Reply:
doesn't understand import.meta yet. There is an active issue to fix this